openapi/components/schemas/GatewayAccountConfig/Ilixium.yaml

```
@@ -45,4 +45,11 @@ allOf:
               - itix
               - tpg
       threeDSecureServer:
-        $ref: ../ThreeDSecureServers/Ilixium3dsServers/Ilixium3dsServers.yaml
+        type: object
+        description: Ilixium 3DS servers.
+        properties:
+          name:
+            description: Merchant plug-in name.
```

suggestion: suggestion for readability (series of nouns)
```suggestion
            description: Name of the merchant plug-in.
```